bin/lib-gm

```diff
@@ -630,21 +655,22 @@ create() {
       EXTRA_WALLETS_COUNTER="$((EXTRA_WALLETS_COUNTER - 1))"
     done
     # Add accounts to genesis
+    GENESIS_CMD_PREFIX=$(get_genesis_cmd_prefix "$1")
     DENOM="$(get_staking_denom "$1")"
-    "$GAIAD_BINARY" add-genesis-account validator "100000000${DENOM},100000000samoleans" --keyring-backend test --home "${HOME_DIR}"
-    "$GAIAD_BINARY" add-genesis-account wallet "100000000${DENOM},100000000samoleans" --keyring-backend test --home "${HOME_DIR}"
+    "$GAIAD_BINARY" $GENESIS_CMD_PREFIX add-genesis-account validator "100000000${DENOM},100000000samoleans" --keyring-backend test --home "${HOME_DIR}"
```

Comment:
Didn't shellcheck complain about this?

Feel free to add a 
```
# shellcheck disable=SC2086
```
on top of it. We explicitly want GENESIS_CMD_PREFIX not quoted since it could be one word or two words.

Reply:
Same in the following 5 cases.

Reply:
added